src/modules/project/pages1/projectView/views/body/sections/header/components/LightningAddress.tsx

```diff
@@ -13,10 +13,9 @@ interface ILightningQR extends ButtonProps {
 
 export const GEYSER_DOMAIN_POSTFIX = '@geyser.fund'
```

Comment:
Shouldn't we define this in the `env` file? Just for convention purposes

Reply:
Not really, this is a variable we add at the end of all geyser lightning addresses, this is not bound to change or a secret, so a normal variable is good enough.